client/src/modules/accounts/accounts.service.js

```diff
@@ -35,9 +50,7 @@ function AccountService(Api, $http, util, bhConstants) {
    *   an array of JSONs.
    */
   function read(id, options) {
-    var url = baseUrl.concat(id || '');
-    return $http.get(url, { params : options })
-      .then(util.unwrapHttpResponse)
+    return Api.read.call(this, id, options)
```

Comment:
Hmmm ... do you really need this? LOL

Reply:
Unfortunately, you do, since it relies needs to do post-processing of the data.  This lets us call `.then()` after the `read()` call.

There might be a prettier way of doing this ... but I'm not sure what it is currently.  Happy to have suggestions if you know it!